hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/ayupov-ayaz/hw01_hello_otus
```

```suggestion
module github.com/ayupov-ayaz/otus-hw-test/hw01_hello_otus
```
Рекомендую указать полный путь до модуля, чтобы он был доступен для скачивания